pkg/stores/partition/store.go

```diff
@@ -60,7 +60,7 @@ func NewStore(partitioner Partitioner, asl accesscontrol.AccessSetLookup) *Store
 		Partitioner: partitioner,
 		asl:         asl,
 	}
-	if v := os.Getenv(cacheDisableEnv); v == "" {
+	if v := os.Getenv(cacheDisableEnv); v == "false" {
 		s.listCache = cache.NewLRUExpireCache(cacheSize)
```

Comment:
Above it says: 

> // Set to non-empty to disable list request caching entirely.

You need to update that, as "false" enables the cache, and it's non empty. 

Also nit: Not sure how public this env var is, but could we have it be a positive instead of a negative ? `cacheEnabled == "false"` with a default of "true" is more intuitive.

Reply:
@cbron I think it makes sense for an env var to be interpreted as false unless it has a set value  of some sort. That is why I would stick with this naming convention, because in the future that should be the behavior.